controllers/polizas.go

```diff
@@ -0,0 +1,97 @@
+
+import (
+	"errors"
+	"strings"
+
+	"github.com/astaxie/beego"
+	"github.com/astaxie/beego/logs"
+	polizasHelper "github.com/udistrital/arka_mid/helpers/polizasHelper"
+)
+
+// PolizasController operations for Polizas
+type PolizasController struct {
+	beego.Controller
+}
+
+// URLMapping ...
+func (c *PolizasController) URLMapping() {
+	c.Mapping("GetAll", c.GetAllElementosPoliza)
+}
+
+// GetAll ...
+// @Title GetAll
+// @Description get AllElementosParaPoliza
+// @Param	query	query	string	false	"Filter. e.g. col1:v1,col2:v2 ..."
+// @Param	fields	query	string	false	"Fields returned. e.g. col1,col2 ..."
+// @Param	sortby	query	string	false	"Sorted-by fields. e.g. col1,col2 ..."
+// @Param	order	query	string	false	"Order corresponding to each sortby field, if single value, apply to all sortby fields. e.g. desc,asc ..."
+// @Param	limit	query	int		false	"Limit the size of result set. Must be an integer"
+// @Param	offset	query	int		false	"Start position of result set. Must be an integer"
+// @Success 200 {object} models.Elemento
+// @Failure 404 not found resource
+// @router /AllElementosPoliza [get]
+func (c *PolizasController) GetAllElementosPoliza() {
+
+	defer func() {
+		if err := recover(); err != nil {
+			logs.Error(err)
+			localError := err.(map[string]interface{})
+			c.Data["mesaage"] = (beego.AppConfig.String("appname") + "/" + "PolizasController" + "/" + (localError["funcion"]).(string))
+			c.Data["data"] = (localError["err"])
+			if status, ok := localError["status"]; ok {
+				c.Abort(status.(string))
+			} else {
+				c.Abort("500") // Error no manejado!
+			}
+		}
+	}()
+
+	var fields []string                 //filtra un valor en concreto
+	var sortby []string                 //
+	var order []string                  //ordenar valores
+	var query = make(map[string]string) //Hace una busqueda de valores
+	var limit int = 10                  //limita la lista para poder paginarla
+	var offset int                      //comienza la lista
+
+	// fields: col1,col2,entity.col3
+	if v := c.GetString("fields"); v != "" {
+		fields = strings.Split(v, ",")
+	}
+	// sortby: col1,col2
+	if v := c.GetString("sortby"); v != "" {
+		sortby = strings.Split(v, ",")
+	}
+	// limit: 10 (default is 10)
+	if v, err := c.GetInt("limit"); err == nil {
+		limit = v
+	}
+	// offset: 0 (default is 0)
+	if v, err := c.GetInt("offset"); err == nil {
+		offset = v
+	}
+	// order: desc,asc
+	if v := c.GetString("order"); v != "" {
+		order = strings.Split(v, ",")
+	}
+	// query: k:v,k:v
+	if v := c.GetString("query"); v != "" {
+		for _, cond := range strings.Split(v, ",") {
+			kv := strings.SplitN(cond, ":", 2)
+			if len(kv) != 2 {
+				c.Data["json"] = errors.New("Error: invalid query key/value pair")
+				c.ServeJSON()
+				return
+			}
+			k, v := kv[0], kv[1]
+			query[k] = v
+		}
+	}
+
+	if l, err := polizasHelper.GetElementosPoliza(offset, limit, fields, order, query, sortby); err != nil {
+		panic(err)
+	} else {
+		c.Data["json"] = l
```

Comment:
Falta el manejo del vacío:
```suggestion
		if l == nil {
			l = []interface{}{}
		}
		c.Data["json"] = l
```
(**No aplicar directamente!**, validar primero y de ser necesario, ajustar de acuerdo al tipo de dato que retorna `polizasHelper.GetElementosPoliza()`)